public/css/style.css

```diff
@@ -1,7 +1,7 @@
 
 :root {
   --color-gold: #D79922;
-  --color-baige: #EFE2BA;
+  --color-beige: #EFE2BA;

```

Comment:
I really need to find a way to get a spell checker into code. I am sure there is a plugin.